x-pack/plugins/ml/public/application/components/field_title_bar/field_title_bar.test.js

```diff
@@ -62,15 +62,16 @@ describe('FieldTitleBar', () => {
     expect(hasClassName).toBeTruthy();
   });
 
-  test(`tooltip hovering`, () => {
+  test(`tooltip hovering`, async () => {
     const props = { card: { fieldName: 'foo', type: 'bar' } };
     const wrapper = mountWithIntl(<FieldTitleBar {...props} />);
     const container = wrapper.find({ className: 'field-name' });
 
     expect(wrapper.find('EuiToolTip').children()).toHaveLength(1);
 
     container.simulate('mouseover');
-    expect(wrapper.find('EuiToolTip').children()).toHaveLength(2);
+    // EuiToolTip mounts children after a 250ms delay
+    setTimeout(() => expect(wrapper.find('EuiToolTip').children()).toHaveLength(2), 250);
```

Comment:
Not sure how `jest` handles this without it, should we add `done()` inside the callback?